### PR TITLE
[PREVIEW] RDM-3126: To show the document uploaded when going back from CYA

### PR DESCRIPTION
**Pull request by MKaramath**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-3126

### Change description ###
To show the selected document is uploaded when going back from CYA

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by MKaramath** [comment]
Added the test cases

File: src/shared/palette/document/write-document-field.component.spec.ts [hunk 1]
```diff
@@ -135,6 +135,12 @@ describe('WriteDocumentFieldComponent', () => {
     expect(uploadElement).toBeTruthy();
   });
 
+  it('should display the selected file name if existsss', () => {
```
Comment (andrewfolga) [inline src/shared/palette/document/write-document-field.component.spec.ts]: Cosmetic but mistypes word in test name.

File: src/shared/components/palette/document/write-document-field.component.ts [hunk 2]
```diff
@@ -131,4 +133,31 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
       this.openFileDialog();
     }
   }
+
+  getSelectedFile() {
+    console.log('1');
+    console.log(this.uploadedDocument);
+    if (this.uploadedDocument) {
+      console.log('Uploaded document exist.');
```
Comment (tzarsmango) [inline src/shared/components/palette/document/write-document-field.component.ts]: could you please remove these console.log lines?

Comment (MKaramath) [reply src/shared/components/palette/document/write-document-field.component.ts]: These console logs added to test/debug on Tony's PC using DM services.  Removed them now.   Done.

**Pull request closed without merging**